src/webextension/injections/js/bug1452707-window.controllers-shim-ib.absa.co.za.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+/**
+ * Bug 1452707 - Build site patch for ib.absa.co.za
+ * WebCompat issue #16401 - https://webcompat.com/issues/16401
+ *
+ * The online banking at ib.absa.co.za detect if window.controllers is a
+ * non-falsy value to detect if the current browser is Firefox or something
+ * else. In bug 1448045, this shim has been disabled for Firefox Nightly 61+,
+ * which breaks the UA detection on this site and results in a "Browser
+ * unsuppored" error message.
+ *
+ * This site patch simply sets window.controllers to a string, resulting in
+ * their check to work again.
+ */
+
+/* globals exportFunction */
+
+Object.defineProperty(window.wrappedJSObject, "controllers", {
+  get: exportFunction(function() {
+    return "window.controllers has been shimmed with a string to get this site's browser detection to work in Firefox 61+.";
```

Comment:
+1 to this approach, seems much safer. I don't know if controllers will be removed in 61 though, so this comment might need to be made more generic or updated once we know what the actual train is.